packages/typespec-client-generator-core/test/package/azure-widget-service.test.ts

```diff
@@ -474,19 +474,19 @@ it("lro delete", async () => {
   await compileAzureWidgetService(
     runnerWithCore,
     `
-      op delete is ResourceOperations.LongRunningResourceDelete<Widget>;
+      op delete is Operations.LongRunningResourceDelete<Widget>;
       `,
   );
   const method = getServiceMethodOfClient(runnerWithCore.context.sdkPackage);
   strictEqual(method.name, "delete");
   strictEqual(method.kind, "lro");
   strictEqual(method.response.type, undefined);
-  strictEqual(runnerWithCore.context.sdkPackage.models.length, 3);
+  strictEqual(runnerWithCore.context.sdkPackage.models.length, 4);
```

Comment:
this now adds the ErrorResponseBase which does get instantiated when we correctly instantiate the interface.

I don't really get what those test are really testing with checking the number of models